src/_guides/language/language-tour.md

```diff
@@ -983,66 +983,36 @@ For more information about maps, see
 [Generics](#generics) and
 [Maps](/guides/libraries/library-tour#maps).
 
-### Runes
+<a id="runes"></a>
+### Runes and grapheme clusters
 
 In Dart, runes are the UTF-32 code points of a string.
+As of Dart 2.6, instead of using runes use the
+[characters package,][characters package]
+which has better support for user-perceived characters,
```

Comment:
@lrhn are there any use cases where `runes` are preferred?

Reply:
When you want to work at the code-point level.

Reasons for doing that could be: 
* Converting the string to UTF-8 or UTF-32.
* Representing code points, like an escaped string,`"\u{10feed}"`, or Unicode code-point notation for debugging, `U+10FEED`.
* Implementing regular expressions (which do not understand grapheme clusters anyway).
* Writing the `characters` package :)

In practice, all of those currently work directly on the UTF-16 code-units to avoid the extra abstraction layer, but they are examples of operations that do work on code points, not grapheme clusters.

Not something most users should worry about.

Reply:
@kwalrath I think it's worthvile explaining the code point part of runes from a positioning view. Maybe something like change:

> In Dart, [runes][] are the Unicode code points of a string. As of Dart 2.6, instead of using runes use the [characters package,][characters package] which has better support for user-perceived characters,

to:

> In Dart, [runes][] expose the Unicode code points of a string. As of Dart 2.6, use the [characters package,][characters package] to view or manipulate user-perceived characters.

Reply:
Done. Ready to go? (I still need to add a test for the code, but that can be in a separate PR.)